packages/math-input/src/components/keypad/operators-page/operators-page.tsx

```diff
@@ -19,22 +27,11 @@ export default class PreAlgebraPage extends React.Component<Props> {
         return (
             <KeypadPageContainer>
                 {/* Row 1 */}
-                <SecondaryKeypadButton
-                    keyConfig={Keys.EXP_2}
-                    onClickKey={onClickKey}
-                />
-                <SecondaryKeypadButton
-                    keyConfig={Keys.EXP}
-                    onClickKey={onClickKey}
-                />
-                <SecondaryKeypadButton
-                    keyConfig={Keys.SQRT}
-                    onClickKey={onClickKey}
-                />
-                <SecondaryKeypadButton
-                    keyConfig={Keys.RADICAL}
-                    onClickKey={onClickKey}
-                />
+                {this.props.preAlgebra ? (
+                    <PreAlgebra onClickKey={onClickKey} />
+                ) : (
+                    <PlaceHolderButton count={4} />
```

Comment:
Is this 4 related to the size of the `PreAlgebra` button set?  Just thinking if there might be a future bug here if we change the `PreAlgebra` component but don't know about this `4`

Reply:
It is. As an alternative, `PreAlgebra` could handle its own placeholders. I'll try that.